components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -768,4 +770,66 @@ - (void)testFullScreenContentLayoutCalculationsComplete {
   // It has no visible side effects, so as long as this test finishes, it passes
 }
 
+- (void)testSettingElevationInDrawerViewControllerPropogatesToPresentationController {
+  // Given
+  self.drawerViewController.elevation = 20;
+
+  // Then
+  MDCBottomDrawerPresentationController *presentationController =
+      (MDCBottomDrawerPresentationController *)self.drawerViewController.presentationController;
+  XCTAssertEqualWithAccuracy(presentationController.elevation, 20, 0.001);
+}
+
+- (void)testCorrectDefaultElevationValue {
+  // When
+  [self.fakeBottomDrawer viewDidLoad];
+
+  // Then
+  XCTAssertEqualWithAccuracy(((MDCShadowLayer *)self.fakeBottomDrawer.shadowedView.layer).elevation,
+                             16, 0.001);
+}
+
+- (void)testElevationIsSetCorrectly {
+  // Given
+  self.fakeBottomDrawer.elevation = 12;
+
+  // When
+  [self.fakeBottomDrawer viewDidLoad];
```

Comment:
I think it's better to call `loadViewIfNeeded`. That should call `viewDidLoad` using the normal lifecycle process.  Is there a reason why `loadViewIfNeeded` cannot be called in these tests?

Reply:
Done.